setup/file_permissions.rst

```diff
@@ -2,7 +2,7 @@ Setting up or Fixing File Permissions
 =====================================
 
 In Symfony 3.x, you needed to do some extra work to make sure that your cache directory
-was writable. But that is no longer true! In Symfony 4, everything works automatically:
+was writable. But that is no longer true! In higher Symfony versions, everything works automatically:
```

Comment:
If this is not true anymore, shall we remove the complete page?

cc @javiereguiluz @wouterj